README.md

```diff
@@ -1,31 +1,9 @@
-# Cardano SL
+# Icarus Importer 
 
-[![Build status](https://badge.buildkite.com/9c3141d21214ff3ea95d0a38a0e1dab59b206159d2841dee44.svg?branch=master)](https://buildkite.com/input-output-hk/cardano-sl)
-[![Windows build status](https://ci.appveyor.com/api/projects/status/github/input-output-hk/cardano-sl?branch=master&svg=true)](https://ci.appveyor.com/project/input-output/cardano-sl)
-[![Release](https://img.shields.io/github/release/input-output-hk/cardano-sl.svg)](https://github.com/input-output-hk/cardano-sl/releases)
+Icarus, a reference implementation for a lightweight wallet developed by the IOHK Engineering Team. This code base can be used as a point of reference to enable developers to create their own secure light and mobile wallets for Cardano. Icarus is a fully open-source code base that will be the first step in a range of open source initiatives to provide developers with a suite of tools to integrate with Cardano.
 
-## What is Cardano SL?
+Icarus Importer connects to will allow wallet users to access blockchain data. For a detailed architecture explanation, please refer to the [wiki](https://github.com/input-output-hk/icarus-poc/wiki/Architecture).
```

Comment:
Typo: `connects to will allow`? Maybe it should be `Icarus Importer will allow wallet`